### PR TITLE
Alena test

### PR DESCRIPTION
**Pull request by AlenaAlexandrovna**
Hi guys, please approve

### CODE REVIEW

**Review by ivash1228 — CHANGES_REQUESTED** [review]
please, delete file from model

**Review by ElenaVasyur — APPROVED** [review]

**Review by ivash1228 — APPROVED** [review]

**Review by ElenaVasyur — APPROVED** [review]

**Review by sweetscripts — APPROVED** [review]
Looks good!

**Review by ivash1228 — APPROVED** [review]

**Review by ElenaVasyur — APPROVED** [review]

**Review by ElenaVasyur — APPROVED** [review]

**Review by ivash1228 — APPROVED** [review]

**Review by sweetscripts — APPROVED** [review]
Good

**Review by ElenaVasyur — APPROVED** [review]

File: src/test/java/school/redrover/CountTotalNumber.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package school.redrover;
+
+public class CountTotalNumber {
```
Comment (aqaestuser) [inline src/test/java/school/redrover/CountTotalNumber.java:3]: class name should end with Test

File: src/test/java/school/redrover/CountTotalNumber.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        public static void main (String[] args) {
```
Comment (aqaestuser) [inline src/test/java/school/redrover/CountTotalNumber.java:4]: main method shouldn't be there at all

**Pull request merged**